### ISSUE TITLE
TAA causes UI and texture jittering 0.0834

### ISSUE THREAD

**Issue by Lafayette028** [LGhassen/Scatterer#158]
With the new default TAA some textures in flight and UI in map screen are jittering. Focus on Apoapsis marker in map view or mountain poking through clouds in flight for an example. Gets worse with lower framerate/frametimes. Disabling TAA and enabling SMAA fixes the issue
[KSP.log](https://github.com/LGhassen/Scatterer/files/8455635/KSP.log)

**Comment by LGhassen** [comment]
Thanks for the report, as said in the forum thread:
```
I don't think it's fixable without breaking the atmospheres in map view with TAA. The thing is TAA works by jittering the projection matrix to take offset samples over time, but the shaders have to work with it.
Could be fixed by overriding the orbit icons shader.
```
 I consider it quite mild though so probably not going to fix it, if someone else wants to though, they can.

**Comment by JonnyOThan** [comment]
The temporal antialiasing from TUFX does not cause the same UI jittering in the space center view, so there must be some way to do this.  It does have other problems in map/tracking station so I’m OK with leaving it off there.

**Comment by JonnyOThan** [comment]
@LGhassen  does jitterSpread correspond to one of the fields in the scatterer GUI?   I tried playing around with them a bit and wasn't really able to find a setting that both didn't cause UI jittering and looked as good as TUFX's implementation.

On a related note, if I disable TAA in scatterer and leave it enabled in TUFX, it seems to get turned off every time I switch to map mode and back.  I haven't done much investigation but my hunch is that scatterer has some kind of logic on scene switch like "if TAA is disabled in my settings, disable it in Unity."  Does that sound accurate?  Should it be opened as a separate issue?

**Comment by LGhassen** [comment]
That issue has already been reported: https://github.com/LGhassen/Scatterer/issues/157

TAA in Scatterer and TUFX are separate so it's not that simple, it's more of a bad interaction.

**Comment by LGhassen** [comment]
Fixed in 4dcbbcc , will be in the next release (0.0836+)
